Dockerfile

```diff
@@ -11,14 +11,22 @@ RUN apt-get -qq update && \
     php7.0-mcrypt php7.0-json php7.0-curl php7.0-zip php7.0-gd \
     pdftk a2ps htmldoc mysql-client \
     php7.0-mbstring \
-    php-xdebug ssh php-common \
+    php-xdebug ssh php-common curl \
     && rm -rf /var/lib/apt/lists/*
```

Comment:
If you remove the apt list further down, you shouldn't remove it here too. apt-get has to retrieve the list again if you do so.